docs/account-management/users/invite-users.md

```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 20
+title: Invite Users
+---
+
+This section describes how to send invitation links to email addresses so new users can access the tenant.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Account**.
+1. Click the **Invite Users** button.
+1. Type the email address to which you want to send an invitation link.
+1. (Optional) Click **Add User** to send additional invitation links, or click the gray icon to remove an email address from the list.
+1. Click **Send Invite**.
+
```

Comment:
I suggest adding a paragraph between the last step and the following paragraph explaining what happens after you click Send Invite. Something like:

Macrometa sends an email to with a signup link to each address that you entered. The links stop working after 24 hours.